src/qt/bitcoingui.cpp

```diff
@@ -734,7 +734,21 @@ void BitcoinGUI::setNumConnections(int count)
     default: icon = ":/icons/connect_4"; break;
     }
     labelConnectionsIcon->setPixmap(QIcon(icon).pixmap(STATUSBAR_ICONSIZE,STATUSBAR_ICONSIZE));
-    labelConnectionsIcon->setToolTip(tr("%1 active connection(s) to Gridcoin network").arg(count));
+
+    if (count == 0)
+    {
+        labelConnectionsIcon->setToolTip(tr("No active connections to the Gridcoin network. "
+                                            "If this persists more than a few minutes, please check your configuration "
+                                            "and your network connectivity.").arg(count));
+    }
+    else if (count == 1)
+    {
+        labelConnectionsIcon->setToolTip(tr("%1 active connection to the Gridcoin network").arg(count));
+    }
+    else
+    {
+        labelConnectionsIcon->setToolTip(tr("%1 active connections to the Gridcoin network").arg(count));
```

Comment:
Qt docs [say this about handling plurals idiomatically](https://doc.qt.io/qt-5/i18n-source-translation.html#handling-plurals):

> ...developers pass an additional integer argument to the [tr()](https://doc.qt.io/qt-5/qobject.html#tr) function, and typically use a special notation for plurals in each translatable string.
>
> If this argument is equal or greater than zero, all occurrences of %n in the resulting string are replaced with a decimal representation of the value supplied. In addition, the translation used will adapt to the value according to the rules for each language.

```suggestion
    else
    {
        labelConnectionsIcon->setToolTip(tr("%n active connection(s) to the Gridcoin network", "", count));
```
...where `%n` is the special placeholder for this format, and localization entries include pluralized variants as needed based on the language.

Reply:
Ok.

Reply:
I tried this. It doesn't appear to work. Maybe we have to update the translations?

Reply:
I am not going to fool with it any more on this PR...

Reply:
Yes, not worth it for this. Just me causing trouble 🙂